docs/docs/mask.md

```diff
@@ -0,0 +1,76 @@
+---
+id: mask
+title: Mask
+sidebar_label: Mask
+slug: /mask
+---
+
+The `Mask` component hides an element by masking the content at specific points.
+Just like its [CSS counterpart](https://developer.mozilla.org/en-US/docs/Web/CSS/mask), there are two modes available:
+* `alpha`: This mode indicates that the transparency (alpha channel) values of the mask layer image should be used as the mask values. This is how masks work in Figma.
+* `luminance`: This mode indicates that the luminance values of the mask layer image should be used as the mask values. This is how masks work in SVG.
+
+The first child of `Mask` is the drawing to be used as a mask, and the remaining children are the drawings to mask.
+
+By default, the mask is not clipped. If you want to clip the mask with the bounds of the content it is masking, use the `bounds` property. 
+
+| Name      | Type                    | Description                                                   |
+|:----------|:------------------------|:--------------------------------------------------------------|
+| mode?     | `alpha` or `luminance` | Is it a luminance or alpha mask (default is `alpha`)          |
+| bounds?   | `SkRect`                | Optional rectangle to clip the mask with                      |
+| children? | `ReactNode[]`           | The first child of `Mask` is the drawing to be used as a mask, and the remaining children are the drawings to mask |
```

Comment:
**Suggestion:**
Correct this to reflect the current model (with components as props)?